### ISSUE TITLE
Dormant <encounterProvider> and <encounterDate> tag

### ISSUE THREAD

**Issue by brightoibe** [nmrs-nigeria/openmrs-module-nmrsmetadata#162]
Forms with multiple providers can be simplified with concepts. This will avoid complicating the data model and making reports slow. e.g

Ordered By   on pharmacy form should be a Concept
Collected By on  Lab Order and Result Form should be a Concept

Let is avoid multiple <encounterProvider> and <encounterDate> tags as much as posible.

**Comment by wandechris** [comment]
@brightoibe why should we avoid multiple tags?, is there a reason for this?

**Comment by brightoibe** [comment]
Yes. They override each other and drive people crazy. <encounterDate> tag at the button of the pharmacy form will overrride the "Visit Date" at the top of the pharmacy form and cause unpredictable effects. They also cause unnessesary data entry budden.

Am proposing that every form have 
1 <encounterDate>, 1 <encounterLoacation> and 1 <encounterProvider> every other element should turn to concepts

**Comment by wandechris** [comment]
@brightoibe what do you mean by 1, 1, and 1?

**Comment by wandechris** [comment]
Also, I'll like us to discuss with a larger house on the fix

**Comment by brightoibe** [comment]
> 
> 
> @brightoibe what do you mean by 1, 1, and 1?

@wandechris  I mean the number of tags. It makes our life easy. Why do we need to capture the name of the data entry person in another <encounterProvider> tag when NMRS stores the current logged in user. Each of this tags create more records on  the encounter_provider table and increase the number of joins. Just my thought.

**Comment by wandechris** [comment]
I don't see any issue with that most especially when data entry usually logs in using the generic admin username, but I'll like us to discuss this in a larger house before we decide.

**Comment by brightoibe** [comment]
Thank you @wandechris our DAs have unique usernames and passwords, also don't forget the POC instance is also ready.

**Comment by wandechris** [comment]
I understand @brightoibe but remember we need to consider other also
